posthog/session_recordings/test/test_session_recording_snapshots_api_blob_v1.py

```diff
@@ -148,6 +142,70 @@ def list_objects_func(path: str) -> list[str]:
             call("an lts stored object path"),
         ]
 
+    @freeze_time("2023-01-01T00:00:00Z")
+    @override_settings(DEBUG=False)
+    @patch("posthog.cloud_utils.is_cloud", return_value=False)
+    @patch(
+        "posthog.session_recordings.queries.session_replay_events.SessionReplayEvents.exists",
+        return_value=True,
+    )
+    @patch("posthog.session_recordings.session_recording_api.object_storage.list_objects")
+    @patch("posthog.session_recordings.session_recording_api.object_storage.get_presigned_url")
+    @patch("posthog.session_recordings.session_recording_api.stream_from", return_value=setup_stream_from())
+    def test_self_hosted_can_still_load_v1_blobs_after_redis_removal(
+        self,
+        _mock_stream_from: MagicMock,
+        mock_presigned_url: MagicMock,
+        mock_list_objects: MagicMock,
+        _mock_exists: MagicMock,
+        _mock_is_cloud: MagicMock,
+    ) -> None:
+        """
+        Smoke test for self-hosted PostHog after Redis realtime removal.
+
+        Self-hosted has blob_v1_sources_are_allowed = True, which means:
+        1. They can list v1 blob sources from S3
+        2. They can fetch those blobs for playback
+        3. No realtime sources should be added (that code was deleted)
+
+        This protects years of historical session recordings for self-hosted customers.
+        """
```

Comment:
ah nice idea.
don't break playback of existing recordings, just stop supporting real time playback at all
cool

i forced the blob_v2 flag for self-hosted a while back as well, so this only really hits folk that have a pretty old install and do a massive jump - and i'd hope most folk will step up through releases to be sure 🙈